### PR TITLE
footer localization

### PR DESCRIPTION
**Pull request by pavelkaly**

### CODE REVIEW

File: data/footerNavigation.ts [hunk 1]
```diff
@@ -3,15 +3,18 @@ export const FOOTER_NAVIGATION = [
...
     description: 'footer.stand_with_ukraine_description',
-    link: 'https://www.stojimezaukrajinou.cz/en',
+    link: 'https://www.stojimezaukrajinou.cz',
+    languages: ['cs', 'uk', 'en'],
   },
   {
     title: 'Česko.Digital',
     description: 'footer.czech_digital',
     link: 'https://cesko.digital',
+    languages: ['en'], //cs variant works only without suffix
```
[the remark this answers is not in the document]
Comment (pavelkaly) [reply data/footerNavigation.ts]: changed by new instructions

File: components/sections/Layout/Footer.tsx [hunk 2]
```diff
@@ -24,9 +25,10 @@ export const Footer = () => {
           </a>
         </div>
         <div className="flex flex-col sm:flex-row justify-between items-center">
-          {FOOTER_NAVIGATION.map(({ title, link, description }, index) => {
+          {FOOTER_NAVIGATION.map(({ title, link, description, languages }, index) => {
+            const languageSuffix = getCountryVariant();
```
[the remark this answers is not in the document]
Comment (pavelkaly) [reply components/sections/Layout/Footer.tsx]: changed by new instructions

File: components/sections/Layout/Footer.tsx [hunk 3]
```diff
@@ -2,9 +2,20 @@ import { useTranslation } from 'next-i18next';
...
 import Image from 'next/image';
 import { FOOTER_NAVIGATION } from 'data/footerNavigation';
+import { getCountryVariant } from 'utils/locales';
+import { useLanguage } from 'utils/useLanguageHook';
 
 export const Footer = () => {
   const { t } = useTranslation();
+  const { currentLanguage } = useLanguage();
+
+  const footerNavigationLinks = Object.entries(FOOTER_NAVIGATION)
```
Comment (juhlarProgress) [inline components/sections/Layout/Footer.tsx:12]: why not just `footerNavigationLinks = FOOTER_NAVIGATION[getCountryVariant()][currentLanguage]`

[the remark this answers is not in the document]
Comment (pavelkaly) [reply components/sections/Layout/Footer.tsx]: changed

**Pull request merged**